docs/index.md

```diff
@@ -42,3 +42,7 @@ dependencies {
 
 Artifacts of SDA Spring Boot Commons are available [at Maven Central](https://search.maven.org/search?q=g:org.sdase.commons.spring.boot)
 since release [0.11.2](https://github.com/SDA-SE/sda-spring-boot-commons/releases/tag/0.11.2).
+
+### Static directory
+Since Spring Boot runs in an embedded Tomcat server, it needs a directory to save its files. Therefore, your application need to set a folder called `static` in the root directory of your container.
+In order to do so, create a folder `static` on `src/main/jib`. When the container is generated with `gradlew jibDockerBuild`, this folder will be copied to the container.
```

Comment:
NIT: Just a formatting suggestion. Having one sentence per line in markdown makes it easier to review changes in the future and to format the code to a reasonable line length. HTML result will be the same.

```suggestion
### Static directory

Since Spring Boot runs in an embedded Tomcat server, it needs a directory to save its files.
Therefore, your application need to set a folder called `static` in the root directory of your container.
In order to do so, create a folder `static` on `src/main/jib`.
When the container is generated with `gradlew jibDockerBuild`, this folder will be copied to the container.
```

I see that it is hard to describe why we need the static folder. But „it needs a directory to save its files“ is actually not true. But currently I do not find a balance of a short explanation that is not too complex and easy to understand and tells the truth.

Maybe you find better words to express that the folder must be there to support readonly file systems (the main reason for the directory but the opposite of „it needs a directory to save its files“).